modules/how-to-plan-your-environment-according-to-cluster-limits.adoc

```diff
@@ -12,6 +12,11 @@ Kubernetes scheduler makes during pod placement. Learn what measures you can
 take to avoid memory swapping.
 ====
 
+====
```

Comment:
Add [IMPORTANT] or [NOTE] before the first ==== to make it  a note box. You could also make this message part of the IMPORTANT note box just before it to avoid having 2 boxes in a row.